tests/test_workers/test_tasks/test_utils.py

```diff
@@ -55,3 +55,17 @@ def test_skopeo_inspect(mock_run_cmd):
     skopeo_args = mock_run_cmd.call_args[0][0]
     expected = ['skopeo', 'inspect', image]
     assert skopeo_args == expected
+
+
+def test_retry():
```

Comment:
Optional: It's nice to keep the ordering of the tests the same as the ordering of the Python module (e.g. alphabetical).

Reply:
hehe, sure :)